README.md

```diff
@@ -33,120 +36,145 @@ is presented below:
+
 ```commandline
 docker run --rm -i --name=macaw_test_container \
 -v <path/to/DrQA/data>:/usr/src/app/DrQA/data \
 -v $("pwd")/stanford-corenlp-full-2017-06-09:/usr/src/app/stanford-corenlp-full-2017-06-09 \
 macaw
 ```
-`</path/to/DrQA/data>` could be `/Users/amitgh/PycharmProjects/DrQA/data` if you downloaded the pre-trained model
-in a separate workspace named DrQA.
+
+`</path/to/DrQA/data>` could be `/Users/amitgh/PycharmProjects/DrQA/data` if you downloaded the pre-trained model in a
+separate workspace named DrQA.
 
 If you did not separately download data at build time, simply run:
+
 ```commandline
 docker run --rm -i --name=macaw_test_container macaw
 ```
 
-In above command we start a container with name *macaw_test_container* from build image *macaw* in interactive mode (`-i`)
-and remove the container when the application exits (`--rm`). After installing all dependencies, it runs `scripts/start.sh`
+In above command we start a container with name *macaw_test_container* from build image *macaw* in interactive
+mode (`-i`)
+and remove the container when the application exits (`--rm`). After installing all dependencies, it
+runs `scripts/start.sh`
 which first starts MongoDB server in a separate thread and then runs `live_main.py`.
 
 #### ssh into the container
-While the application is running, we can go inside the container to see the contents (directory structure, indri index, etc.).
+
+While the application is running, we can go inside the container to see the contents (directory structure, indri index,
+etc.).
+
 ```commandline
 docker exec -it macaw_test_container /bin/bash
 ```
 
 #### Updating TREC data for Indri
-Indri index is created using the document stored in `trec_documents/` directory. It has some default data. To create a bigger
-index, download the entire data from [archive](https://archive.org/details/trec-ir) and put it in trec_documents. Docker will 
-copy it during build time and create a new index. Index creation parameters like *-memory*, *-stemmer.name* can be changed
-inside the Dockerfile.
+
+Indri index is created using the document stored in `trec_documents/` directory. It has some default data. To create a
+bigger index, download the entire data from [archive](https://archive.org/details/trec-ir) and put it in trec_documents.
+Docker will copy it during build time and create a new index. Index creation parameters like *-memory*, *-stemmer.name*
+can be changed inside the Dockerfile.
 
 ### A note on dependencies
-Pyndri is tested only with [Indri-5.11](https://sourceforge.net/projects/lemur/files/lemur/indri-5.11/) which in turn only
-supports certain OS. We use Ubuntu-16.04 in our Docker container. Ubuntu 18.04 creates issues during indri index creation
-because of some unsupported C++11 functions. *pip* version <21.0 is needed to properly install pyndri.
 
+Pyndri is tested only with [Indri-5.11](https://sourceforge.net/projects/lemur/files/lemur/indri-5.11/) which in turn

```

Comment:
Pyndri is no longer needed so you can modify this dependency note. Add any new things that are still relevant.